### DISCUSSION TITLE
Correct set-up with pictures on a windows network share running photoprism in docker (linux)?

### DISCUSSION THREAD

**Discussion by DSew20** [photoprism/photoprism#1900] [Q&A] [answered]
Hi,

I would imagine this is common, but I've struggled and searched for guidance on how to add a windows share (e.g. \\\server\pictures) to photoprism/originals in the docker-compose.

Does anybody know of a walkthrough on how to do this?

I am using the latest docker-compose for Photoprism, which includes mariadb.  Fires up ok with the default "~/Pictures:/photoprism/originals" volume.  But, I can not find a simple way to add a network share.

**Comment by DSew20** [comment, marked as answer]
I seem to have got this working.

I added the line to fstab:

//192.168.x.yyy/Pictures /mnt/pictures cifs username=<xxxxxxx>,password=<yyyyyyy>,vers=2.0 0 0

I can now index the pictures on the network.  Hope this helps others.

If there is a better way, please tell.

**Reply by graciousgrey** [reply]
See https://docs.photoprism.app/getting-started/faq/#how-can-i-mount-network-shares-with-docker for instructions
